src/core/InputDropdown/style.ts

```diff
@@ -48,7 +48,7 @@ const inputDropdownStyles = (props: InputDropdownProps): SerializedStyles => {
         color: #000;
       }
 
-      .styled-label {
```

Comment:
Are we able to remove `styled-label` from classes prop now that we can use component selector 🙏 ?

Thank you!

Reply:
Aah yeah, for sure, I forgot about that. 😵‍💫 
I will update the PR